service-a/src/main/java/com/example/services/CustomerServiceAlternativeClient.java

```diff
@@ -0,0 +1,16 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package com.example.services;
+
+import org.springframework.cloud.netflix.feign.FeignClient;
+
+/**
+ *
+ * @author roland.werner
+ */
+@FeignClient(value = "serviceB" , fallback = ServiceBClientFallback.class)
+public interface CustomerServiceAlternativeClient extends CustomerService {
```

Comment:
Du könntest das auch direkt in Interface schreiben:

```java
@FeignClient(value = "serviceB" , fallback = ServiceBClientFallback.class)
public interface CustomerServiceAlternativeClient {

  @RequestMapping(value = "/customer/{id}", method = RequestMethod.GET, produces = "application/json")
  Customer getCustomer(@PathVariable("id") int id);
}
```
Gibt es einen Grund, warum du hier ableitest?